Git-Remove-Submodule.md

```diff
@@ -0,0 +1,16 @@
+Removing a submodule is useful when it is no longer required. The steps below outline the removal of a submodule.
```

Comment:
Might want to link to what a [submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules) is.